CHANGELOG.md

```diff
@@ -1,266 +1,209 @@
-| Fixed a Service Worker HTML caching bug                                                 | **Bugfix**  | [#2390][] |
-| Fixed an incorrect `propType` in order confirmation page                                | **Bugfix**  | [#2375][] |
-| Fixed a bug where Magento2 media image would not load                                   | **Bugfix**  | [#2339][] |
-| Fixed checkout button props                                                             | **Bugfix**  | [#2334][] |
-| Added a white stroke to the checkout button css in Mini Cart                            | **Bugfix**  | [#2330][] |
-| Fixed bug where a scroll position would not reset on router change                      | **Bugfix**  | [#2309][] |
-| Fixed a css var in the checkbox.css                                                     | **Bugfix**  | [#2307][] |
-| Fixed a rounding error on image src set                                                 | **Bugfix**  | [#2300][] |
-| Fixed font size on the Order Summary page                                               | **Bugfix**  | [#2275][] |
-| Removed a duplicate css property                                                        | **Bugfix**  | [#2248][] |
-| Fixed the placement of next and previous button for the Carousel component in Chrome    | **Bugfix**  | [#2512][] |
+| Description                                                                          | Change type  | PR        |
+| ------------------------------------------------------------------------------------ | ------------ | --------- |
+| Implemented initial code for the new MiniCart component                              | **Feature**  | [#2494][] |
+| Created new components for an account menu                                           | **Feature**  | [#2550][] |
+| Created components for a Newsletter Subscription page                                | **Feature**  | [#2571][] |
+| Implemented initial code for an Order History page                                   | **Feature**  | [#2611][] |
+| Implemented a way to access top/intermediate level categories through the left nav   | **Feature**  | [#2616][] |
+| Implemented initial UX and workflow for a Forgot Password feature for My Account     | **Feature**  | [#2619][] |
+| Implemented initial code for Wishlist                                                | **Feature**  | [#2620][] |
+| Implemented the ability to translate Venia's header / footer                         | **Feature**  | [#2643][] |
+| Added ability to allow PWA Studio to use a different store to localize CMS Pages     | **Feature**  | [#2649][] |
+| Implemented initial code for Address Book                                            | **Feature**  | [#2653][] |
+| Implemented initial code for an Order History table                                  | **Feature**  | [#2660][] |
+| Updated Venia's button style to match new design                                     | **Update**   | [#2496][] |
```

Comment:
Would it be easier if we added a "Change Type" label on our PRs like we have with versions?

Feature, Update, Refactor, Bugfix?

Reply:
yeah that would help